### DISCUSSION TITLE
New LD-006 user needs help with Exposure testing

### DISCUSSION THREAD

**Discussion by Rosinstein** [sn4k3/UVtools#647] [Q&A] [answered]
Hello all, I've been following this video https://youtu.be/lRbZw6GbGCI and unfortunately my flash drive was curropt and unsuable.  I need a presliced file or at least think I do.  Im just trying to get the exposure test done so that I can move forward reliably.  Thanks in advace for you time and consideration.   EDIT: I was able to get my files with a FB group!

**Comment by sn4k3** [comment, marked as answer]
You need to slice one in your slicer with your settings for that printer. Asking files from another source is always bad no matter what you want to do with it.
